tests/models/cohere/test_modeling_cohere.py

```diff
@@ -393,6 +393,7 @@ def test_batched_4bit(self):
         output = model.generate(**inputs, max_new_tokens=40, do_sample=False)
         self.assertEqual(tokenizer.batch_decode(output, skip_special_tokens=True), EXPECTED_TEXT)
 
+    @require_torch_sdpa
```

Comment:
It looks like the test about should have `require_bitsandbytes` too 👀 

@ydshieh If we have this requirement, will this test be run as part of the slow overnight tests?

Reply:
@amyeroberts I pushed a commit to add the `require_bitsandbytes` flag for `test_batched_4bit`, `test_batched_small_model_logits` is fine without bitsandbytes.